src/commands/main.js

```diff
@@ -6,17 +6,17 @@ const { system } = require('../utils')
 const version = require('../../package.json').version
 
 module.exports = async function (message, parent) {
-  let summary = `Dokdo v${version}, discord.js \`${parent.Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at ${dayjs(system.processReadyAt()).fromNow()}, bot was ready at ${dayjs(parent.client.readyAt).fromNow()}.\n`
+  let summary = `Dokdo \`v${version}\`, discord.js \`${parent.Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at \`\`${dayjs(system.processReadyAt()).fromNow()}\`\`, bot was ready at \`\`${dayjs(parent.client.readyAt).fromNow()}\`\`.`
```

Comment:
```suggestion
  let summary = `Dokdo v${version}, discord.js \`${parent.Discord.version}\`, \`Node.js ${process.version}\` on \`${process.platform}\`\nProcess started at ${dayjs(system.processReadyAt()).fromNow()}, bot was ready at ${dayjs(parent.client.readyAt).fromNow()}.\n`
```

Please do not make change.